docs/docs/config/system.md

```diff
@@ -135,3 +135,53 @@ Following [split keyboard](../features/split-keyboards.md) settings are defined
 | `CONFIG_ZMK_SPLIT_BLE_PERIPHERAL_STACK_SIZE`            | int  | Stack size of the BLE split peripheral notify thread                       | 650                                        |
 | `CONFIG_ZMK_SPLIT_BLE_PERIPHERAL_PRIORITY`              | int  | Priority of the BLE split peripheral notify thread                         | 5                                          |
 | `CONFIG_ZMK_SPLIT_BLE_PERIPHERAL_POSITION_QUEUE_SIZE`   | int  | Max number of key state events to queue to send to the central             | 10                                         |
+
+## Snippets
+
+Snippets are a way to save common configuration separately when it applies to multiple different applications.
+
+More documentation is available from [Zephyr](https://docs.zephyrproject.org/3.5.0/build/snippets/index.html).
+
+Enable snippets with `-S <snippet>` for local builds, for example:
+
+```sh
+west build -b nrfmicro_13_52833 -S nrf52833-nosd -- -DSHIELD=corne_left
+```
+
+Or by adding `snippet: <snippet>` to your `build.yaml` for the appropriate board:
+
+```yaml
+- board: nrfmicro_13_52833
+  snippet: nrf52833-nosd
+  shield: corne_left
+```
```

Comment:
````suggestion
Enable snippets by adding `snippet: <snippet>` to your `build.yaml` for the appropriate board:

```yaml
- board: nrfmicro_13_52833
  snippet: nrf52833-nosd
  shield: corne_left
```

For local builds, add `-S <snippet>` to your build command. For example:

```sh
west build -b nrfmicro_13_52833 -S nrf52833-nosd -- -DSHIELD=corne_left
```
````
Swapped order because we tend to recommend GHA first.

Reply:
Swapped, thank you. I tend to focus on local builds for development.